### DISCUSSION TITLE
Random 502 bad gateway errors in AKS Cluster

### DISCUSSION THREAD

**Discussion by Bisharsh** [kedacore/http-add-on#1055] [Q&A]
Tried using HTTP keda add-on for scaling in our Azure Kubernetes Service (AKS) Cluster, however, started seeing multiple 502 bad gateway errors. Though they are not causing any downtime but in the datadog logs we see continuous 502 errors which are random in nature. Therefore, not able to reproduce the error. 

Did anyone face any similar issues?

**Comment by thisurawijesekera** [comment]
Did you found any cause or solution for this?

**Reply by widdix123** [reply]
Anyone knows how to fix this ?

**Reply by hari10599** [reply]
https://github.com/kedacore/charts/blob/61b9fc22f64e36f9dd4bd4e260b64eb2f6f98b0c/http-add-on/values.yaml#L165 -> Adjust this value according to your requirements
